lib/dates.rb

```diff
@@ -0,0 +1,38 @@
+require 'date'
+require 'pry'
+
+class Dates
+  attr_reader :date, :date_range
+
+  def self.date_format(date)
```

Comment:
Calling this class `Dates` when Ruby has a built-in `Date` class could get confusing fast. Something more visually distinct like `DateRange` might be a better choice.